crates/telio-model/src/config.rs

```diff
@@ -26,6 +27,17 @@ pub struct PeerBase {
     pub nickname: Option<String>,
 }
 
+impl fmt::Debug for PeerBase {
```

Comment:
I think that wrapping private fields in `telio_utils::Hidden` is less error prone. It's harder to print log them accidentally and you don't have to write custom debug impls :)